hw/mcu/stm/stm32_common/src/hal_uart.c

```diff
@@ -24,6 +24,7 @@
 #include "bsp/bsp.h"
 #include <assert.h>
 #include <stdlib.h>
+#include "os/mynewt.h"
```

Comment:
It seems strange that this file missed include file (please confirmed)

Reply:
I confirm this file did not include os/mynewt.h (and hence does not compile if NDEBUG is defined)